lib/tasks/dev.rake

```diff
@@ -1,33 +1,30 @@
 desc "Fill the database tables with some sample data"
 task({ :sample_data => :environment }) do
+  starting = Time.now
```

Comment:
I'm not sure that this line is necessary for your sample data task. Feel free to follow-up with me during class if you have questions!